platform/pulp/app/viewsets/catalog.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import pagination
+
+from pulp.app.models import DownloadCatalog
+from pulp.app.serializers import DownloadCatalogSerializer
+from pulp.app.viewsets import NamedModelViewSet
+
+
+class DownloadCatalogPagination(pagination.CursorPagination):
```

Comment:
I think the pagination should be moved to [here](https://github.com/pulp/pulp/blob/3.0-dev/platform/pulp/app/pagination.py)